package.json

```diff
@@ -15,6 +15,7 @@
     "cypress:test": "lerna run cypress:test",
     "commit": "git-cz",
     "bump:version": "node ./scripts/deploy/bump-version.js",
+    "version": "npm run lockLernaDependencies && git add \"**/package-lock.json\"",
```

Comment:
where does it get told to use this script in the `version` lifecycle? is it automatic by naming this script the same as the lifecycle name?